GREEDY.UnitTests/WritingToFileControllerUnitTest.cs

```diff
@@ -3,33 +3,90 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.IO;
+using System.Configuration;
+//using Moq;
+using GREEDY;
+using GREEDY.Interfaces;
+using Telerik.JustMock;
 
 namespace GREEDY.UnitTests
 {
     [TestClass]
     public class WritingToFileControllerUnitTest
     {
+
+        private const string _GoodFileName = "TestReceipt.txt";
+        private const string _BadFileName = "";
+
         [DataTestMethod]
+        [DataRow("1545")]
+        [DataRow("     ")]
         [DataRow("")]
-        [DataRow("   ")]
-        [DataRow("123")]
-        public void WriteToFileTest (string data)
+        [DataRow("123456 cdcd s")]
+
+        public void WriteTextToGoodFile (string data)
         {
-            //arrange
+            //Arrange
```

Comment:
We should keep to one standard (if arrange, act, assert were written in lower case until now we should keep it that way).